internal/service/alertconfiguration/resource_alert_configuration.go

```diff
@@ -261,7 +261,7 @@ func (r *alertConfigurationRS) Schema(ctx context.Context, req resource.SchemaRe
 						"datadog_region": schema.StringAttribute{
 							Optional: true,
 							Validators: []validator.String{
-								stringvalidator.OneOf("US", "EU"),
+								stringvalidator.OneOf("US", "EU", "US3", "US5"),
```

Comment:
@marcosuma thinking more long term, we could remove these client-side validations entirely if the API is returning proper error messages for incorrect values. (while also having the documentation reference the Atlas API)

Reply:
I agree on removing the validation on the provider for fields that have multiple values (3 + and we know will have more values in the future). We did the same for the cluster region field

Reply:
I am ok with that, @AgustinBettati @andreaangiolillo do we leave the documentation with the values though?

Reply:
I don't think we should. We would be force to update it all the time that the endpoint add/remove a value

Reply:
Agree, this is an example https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/website/docs/r/search_deployment.html.markdown?plain=1#L44 but we have several others in our docs.

Reply:
FYI @jfmainville - tl;dr: I think we'd prefer to remove this validation completely and update the documentation by referencing Datadog API supported values.

We can do two things:
* you make these changes
* we cancel this PR and we make the change on our side 

Either way should work, FYI @AgustinBettati @andreaangiolillo

Reply:
@marcosuma I can make the changes, no issues, I'll remove the validation and add a link to the MongoDB Atlas API that shows the supported values for the Datadog regions (to have a reference). Thanks!

Reply:
Also, to make sure that I comply with the appropriate structure for the documentation, is this line valid for the `datadog_region` attribute?

`* `datadog_region` - Region that indicates which API URL to use. See the `notifications` body parameter in the [MongoDB API Alert Configuration documentation](https://www.mongodb.com/docs/atlas/reference/api-resources-spec/v2/#tag/Alert-Configurations/operation/createAlertConfiguration) for the details. The default Datadog region is US.`

Reply:
Thank so much @jfmainville, we all really appreciate your commitment here, and sorry for late reply. I would say something like:

```
`datadog_region` - Region that indicates which API URL to use. See the `datadogRegion` field in the `notifications` request parameter of [MongoDB API Alert Configuration documentation](https://www.mongodb.com/docs/atlas/reference/api-resources-spec/v2/#tag/Alert-Configurations/operation/createAlertConfiguration) for more details. The default Datadog region is US.
```

checking in with @andreaangiolillo @AgustinBettati so to make sure you don't have to do other changes later and we agree on it here.

Reply:
SGTM, thank you @jfmainville for following up on this.

Reply:
Awesome, thanks for the details! I updated the codebase accordingly, let me know if additional changes are required. Thanks!